corehq/apps/reminders/dbaccessors.py

```diff
@@ -0,0 +1,9 @@
+def get_surveys_in_domain(domain):
+    from corehq.apps.reminders.models import Survey
+    return sorted(Survey.view(
```

Comment:
looking below, it doesn't seem that the get sorted, any reason for doing so? probably not a big deal